x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/hooks/use_fetch_agents_data.tsx

```diff
@@ -22,13 +22,15 @@ import {
   sendGetAgentTags,
   sendGetAgentPolicies,
   useAuthz,
+  sendGetActionStatus,
 } from '../../../../hooks';
 import { AgentStatusKueryHelper, ExperimentalFeaturesService } from '../../../../services';
 import { AGENT_POLICY_SAVED_OBJECT_TYPE, SO_SEARCH_LIMIT } from '../../../../constants';
 
 import { getKuery } from '../utils/get_kuery';
 
-const REFRESH_INTERVAL_MS = 30000;
+const REFRESH_INTERVAL_MS = 10000;
```

Comment:
Is it necessary to refresh every 10s? I would keep the 30s to keep the API requests down.

Reply:
Oops, I changed that for testing only! Sorry, reverting.